README.md

```diff
@@ -1,2 +1,78 @@
-# jboss-eap-openshift-templates
-ImageStreams and Templates for the OpenShift Images of JBoss EAP
+# Red Hat JBoss Enterprise Application Platform XP 1.0 OpenShift container image imagestreams and templates
```

Comment:
I created the file as markdown. Could you please rename it to `README.adoc`?

Reply:
Yep, sorry I didn't see that one.